src/emulator/functionsEmulator.ts

```diff
@@ -1119,6 +1156,11 @@ export class FunctionsEmulator implements EmulatorInstance {
       process.env.PUBSUB_EMULATOR_HOST = pubsubHost;
     }
 
+    const eventarcEmulator = this.getEmulatorInfo(Emulators.EVENTARC);
+    if (eventarcEmulator) {
+      process.env.CLOUD_EVENTARC_EMULATOR_HOST = `http://${formatHost(eventarcEmulator)}`;
```

Comment:
Should this be CLOUD_EVENTARC_EMULATOR_HOST or EVENTARC_EMULATOR_HOST?

Reply:
CLOUD_EVENTARC_EMULATOR_HOST, resolved in sync!